package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Memory leak testing for node. Write tests using your favorite test runner.",
   "main": "lib/index.js",
   "scripts": {
-    "test": "standard lib/**/*.js"
+    "test": "node_modules/.bin/standard lib/**/*.js && node_modules/.bin/mocha test"
```

Comment:
npm adds `node_modules/.bin` to `PATH`, so you can use `standard` and `mocha` without the manual path prefix